Flat/src/ModulesLepPho.cc

```diff
@@ -335,6 +335,80 @@ void PandaAnalyzer::Photons()
     tr->TriggerEvent("photons");
 }
 
+void PandaAnalyzer::ComplicatedPhotons()
+{
+    for (auto& pho : event.photons) {
+      if (!pho.medium)
+        continue;
+      float pt = pho.pt() * EGMSCALE;
+      if (pt<1) continue;
+      float eta = pho.eta(), phi = pho.phi();
+      if (pt<25 || fabs(eta)>2.5)
+        continue;
+      if (IsMatched(&matchLeps,0.16,pho.eta(),pho.phi()))
+        continue;
+      loosePhos.push_back(&pho);
+      gt->nLoosePhoton++;
+      if (gt->loosePho1Pt < pt) {
+        gt->loosePho1Pt = pt;
+        gt->loosePho1Eta = eta;
+        gt->loosePho1Phi = phi;
+        int phoSelBit = 0;
+	if(pho.medium)                 phoSelBit |= pMedium; // this is always true as of now, but safer to have it like this
+	if(pho.tight)                  phoSelBit |= pTight;
+	if(pho.highpt)                 phoSelBit |= pHighPt;
+	if(pho.csafeVeto)              phoSelBit |= pCsafeVeto;
+	if(pho.pixelVeto)              phoSelBit |= pPixelVeto;
+	if(!PFChargedPhotonMatch(pho)) phoSelBit |= pTrkVeto;
```

Comment:
whitespace/indentation is messed up

Reply:
For some reason I see it well on my editor, sorry for that